docs/Tutorial_Advanced_Annotation_Vectors.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<p>I've uploaded the data to zenodo: https://zenodo.org/record/5067773/files/electrocardiography.csv?download=1</p><p> </p><p>Note that the column name name has been changed to "Electric Potential" (instead of "Electrical") </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>